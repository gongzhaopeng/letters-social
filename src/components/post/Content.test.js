import Content from "./Content";

describe("<Content/>", () => {
    test("should render correctly", () => {

    });
});
