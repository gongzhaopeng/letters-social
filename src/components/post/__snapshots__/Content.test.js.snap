// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Content/> render methods snapshot 1`] = `
<p
  className="content"
>
  I am learning to test React components
</p>
`;
